src/platform_impl/macos/observer.rs

```diff
@@ -169,31 +179,50 @@ impl Default for EventLoopWaker {
                 ptr::null_mut(),
             );
             CFRunLoopAddTimer(CFRunLoopGetMain(), timer, kCFRunLoopCommonModes);
-            EventLoopWaker { timer }
+            EventLoopWaker {
+                timer,
+                start_instant: Instant::now(),
```

Comment:
I've removed the `-` because it was crashing, since you need to pass 1000 seconds from the boot to make it work, which I usually don't, the clock is in monotonic anyway, so if you call `::now` later, this one will be in the past.

Reply:
Ah, yeah, that makes sense